src/cpp/src/ket/KetPair.cpp

```diff
@@ -29,13 +29,18 @@ KetPair<Scalar>::KetPair(
 template <typename Scalar>
 std::string KetPair<Scalar>::get_label() const {
     std::string label;
-    std::string separator;
```

Comment:
I would keep the separator trick, as we also use it in other parts of the C++ code.